### PR TITLE
fix: hoist vitest import

### PR DESCRIPTION
**Pull request by sheremet-va**
Related: #425

### CODE REVIEW

**Comment by antfu** [comment]
Can you add some tests?

**Comment by sheremet-va** [comment]
> Can you add some tests?

Done

File: test/core/test/hoist-import.test.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { expect, test, vi } from 'vitest'
+// @ts-ignore
```
Comment (antfu) [inline test/core/test/hoist-import.test.ts:2]: What are we ignoring?

Comment (sheremet-va) [reply test/core/test/hoist-import.test.ts:2]: timeout doesn't have fn, mock adds it

File: test/core/test/hoist-import.test.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { expect, test, vi } from 'vitest'
+// @ts-ignore timeout doesn't have fn, mock factory adds it
```
Comment (antfu) [inline test/core/test/hoist-import.test.ts:2]:
```suggestion
// @ts-expect-error timeout doesn't have fn, mock factory adds it
```

**Pull request merged**